corehq/apps/users/models.py

```diff
@@ -1139,6 +1139,33 @@ def get(stale, raise_if_none):
         else:
             return None
 
+    @classmethod
+    @skippable_quickcache(['username'], skip_arg='strict')
+    def get_deleted_by_username(cls, username, strict=True):
+        def get(stale, raise_if_none):
+            result = cls.get_db().view('deleted_users/by_username',
+                                       key=username,
+                                       include_docs=True,
+                                       reduce=False,
+                                       stale=stale if not strict else None,
+                                       )
+            return result.one(except_all=raise_if_none)
```

Comment:
So one raises an exception in case of more matches. But it didn't except limit as an argument. 
`TypeError: one() got an unexpected keyword argument 'limit'`
Using first instead of one. That handles the case.